keyboards/ploopyco/madromys/config.h

```diff
@@ -18,6 +18,8 @@
 
 #pragma once
 
+#define VIA_FIRMWARE_VERSION 1
```

Comment:
@light-idea Thanks for your suggestion to version bump VIA. Does this have the intended effect? I followed the commit you linked in your comment.

I was also wondering about your choice of `config.h`. You chose to modify `.../ploopyco/trackball/rev1_005/config.h` instead of `.../ploopyco/trackball/config.h`. I think defining it in either should have the intended effect. What considerations are there with this?